narwhals/expression.py

```diff
@@ -2290,7 +2290,43 @@ def min(*columns: str) -> Expr:
 
 def max(*columns: str) -> Expr:
     """
-    Instantiate an expression representing the maximum of one or more columns, similar to `polars.max`.
+    Return the maximum value.
+
+    Note:
+       Syntactic sugar for nw.col(names).max().
+
+    Parameters:
+        *names
```

Comment:
the function signature says `columns`, but here you have `names` - can you update that to also use `names`?

(we should do the same with other such functions)